### ISSUE TITLE
Accessibility: Slideshow icons have no alternative text

### ISSUE THREAD

**Issue by swashbuck** [adaptlearning/adapt-contrib-narrative#319]
### Subject of the issue
We have recently had an accessibility audit for one of our clients. The consulting agency flagged the following issue regarding the progress dots underneath Narrative images.

> The slideshow component has dot indicators below to inform users:
> 1. how many slides are present
> 2. which slide they are on
>
> However, they do not have a text alternative to describe its purpose.
>
> Screen reader users who rely on text alternatives to understand the purpose of non- text content will not be provided with this information.

Their recommendation:
> Ensure that all meaningful icons have the text alternatives available. In this case, the dots are marked up using background colours.
>
> * Add role="img" to the container of the icons
> * Add an informative aria-label to accurately describe its purpose ◦ Update the aria-label when the slide position changes
>
> Code example:
```
<div class="narrative__indicators narrative__slide-indicators" role="img" aria-label="Slide 1 of 2"></div>
```

**Type:** WCAG 2
**Principle:** Perceivable
**Level:** A
**Severity:** High
**Guideline:** 1.1 Text Alternatives
**Success Criteria:** 1.1.1 Non-text Content (A)

### Screenshots
<img width="573" alt="Screenshot 2025-01-14 at 2 59 26 PM" src="https://github.com/user-attachments/assets/533f1a9e-4adf-4904-8c8b-837aaabca07a" />

**Comment by oliverfoster** [comment]
The buttons have alternate labels with `(item {{itemNumber}} of {{totalItems}})`:

https://github.com/adaptlearning/adapt-contrib-narrative/blob/a31a64bfdc8d99821c89c58b1167f2560060e1af/properties.schema#L15-L30

Here is a summary of why those dots are not accessible:
https://github.com/adaptlearning/adapt-contrib-narrative/issues/273#issuecomment-1653898528

**Comment by swashbuck** [comment]
Thanks, @oliverfoster . I'll include this in our response back to the a11y consultants.
